apps/deno/index.ts

```diff
@@ -150,7 +151,7 @@ const main = async () => {
   // prettier-ignore
   const allTests = [
       test1, test2, test3, test4, test5, test6, test7, test8, test9, test10,
-      test11, test12, test13
+      test11, test12, test13, test14
```

Comment:
Instead of creating a new test, can we update some of the drawing operations for existing tests (or add a page if necessary)? The integration tests cannot easily be automated and they take a lot of time to run on all the platforms prior to each release, so I try to keep the total number of tests to a minimum. I prefer to exercise as much functionality in as few tests and pages as possible.